nemo_aligner/experimental/self_rewarding/conf/gpt_self_rewarding_llama3.yaml

```
@@ -0,0 +1,282 @@
+
+    # Current default meta-judge prompt is copied from https://arxiv.org/abs/2407.19594
+    llm_meta_judge_prompt: |
+      ${model.data.chat_prompt_tokens.system_turn_start}<|start_header_id|>system<|end_header_id|>
+      
+      ${model.data.chat_prompt_tokens.end_of_turn}<|start_header_id|>user<|end_header_id|>${model.data.chat_prompt_tokens.end_of_name}
+      Review the user's question and the corresponding response, along with two judgments.
+      Determine which judgment is more accurate according to the rubric provided below. The
+      rubric used for the initial judgments is as follows:
+      - Add 1 point if the response is relevant and provides some information related to
+      the user's inquiry, even if it is incomplete or contains some irrelevant content.
+      - Add another point if the response addresses a substantial portion of the user's question,
+      but does not completely resolve the query or provide a direct answer.
+      - Award a third point if the response answers the basic elements of the user's question in a
+      useful way, regardless of whether it seems to have been written by an AI Assistant or if it
+      has elements typically found in blogs or search results.
+      - Grant a fourth point if the response is clearly written from an AI Assistant's perspective,
+      addressing the user's question directly and comprehensively, and is well-organized and helpful,
+      even if there is slight room for improvement in clarity, conciseness or focus.
+      - Bestow a fifth point for a response that is impeccably tailored to the user's question
+      by an AI Assistant, without extraneous information, reflecting expert knowledge, and
+      demonstrating a high-quality, engaging, and insightful answer.
+      
+      <prompt>{{ prompt }}<prompt>
+      <response>{{ response }}<response>
+      <judgement A>{{ judgement_a }}<judgement A>
+      <judgement B>{{ judgement_b }}<judgement B>
+      
+      After examining the original question, response, and both judgments:
+      - Explain which judgment is more accurate according to the original rubric and why.
+      Consider factors such as adherence to the rubric, accuracy in evaluating the response, and
+      consistency in applying the criteria.
+      - Conclude with a clear statement of which judgment is better using the format: "Winner: [Judgement A | Judgement B]"
+      ${model.data.chat_prompt_tokens.end_of_turn}<|start_header_id|>assistant<|end_header_id|>${model.data.chat_prompt_tokens.end_of_name}
+
+
+    # params for generation
+    sampling_params:
+      use_greedy: False
+      temperature: 1.0
+      top_k: 0
+      top_p: 1.0
+      repetition_penalty: 1.0
+      add_BOS: False
+      all_probs: False
+      compute_logprob: False
+      end_strings: ["<|endoftext|>", "<extra_id_1>", "<|eot_id|>"]
```

Should be safe (and potentially more efficient) to get rid of `<extra_id_1>` for Llama3
```suggestion
      end_strings: ["<|endoftext|>", "<|eot_id|>"]
```

---

Fixed